doc/release-notes/rl-2505.section.md

```diff
@@ -48,7 +48,7 @@
 
 ### NexusMods.App upgraded {#sec-nixpkgs-release-25.05-incompatibilities-nexusmods-app-upgraded}
 
-- `nexusmods-app` has been upgraded from version 0.6.3 to 0.8.2.
+- `nexusmods-app` has been upgraded from version 0.6.3 to 0.8.3.
```

Comment:
We could probably put a more general version number here to avoid changing it repeatedly.

Reply:
Maybe. Not sure how I'd phrase it though.

I'd expect we only have a few more releases until 25.05 branches off anyway, so I don't see it as a major issue.

I believe upstream's _usual_ release schedule is roughly every 3 weeks or so.

Reply:
based on this:

> When upgrading from version 0.7.0 or newer, it should be safe to upgrade without resetting/uninstalling.

I'd probably just say 0.7.0+ or something.

Really not a big deal, it's just something that's likely to be forgotten anyway, e.g. with an automated update PR.